workgroup/pom.xml

```diff
@@ -30,7 +30,7 @@
         <dependency>
             <groupId>software.amazon.awssdk</groupId>
             <artifactId>athena</artifactId>
-            <version>2.10.40</version>
+            <version>2.16.4</version>
```

Comment:
Try to make sure this update is good.

Reply:
it was the latest that had the workgroup + engine version release